README.md

```diff
@@ -3,6 +3,10 @@
 Various Janet utility modules. Spork aims to be grab bag of useful Janet functionality that
 does not belong in the core library.
 
+To use all features mentioned bellow in ones program you can do `(use spork)` to
```

Comment:
s/bellow/below/ and s/ones/one's/ ?

Reply:
This is the sentence I added myself :facepalm: